README.md

```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
 
 ## How the samples are organized
 
-The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models. Note that the support for reliable stateful service is not there today and will come in near future.
-
-## Actor samples
+The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models.
+
+### Folder Hierarchy
+src/ - Source of the application divided by different modules by sub-folders.  
+&lt;Application Folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are places in code   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subfolders.  
```

Comment:
Instead of adding these spaces, should we use bullet/ hyphen for the items on left?

Reply:
The spaces are to align the multi-line explanation. Adding bullet won't help in this case.
I can add them for all the folders though to make it look more clean. Let me try.